Python/Product/Django/Intellisense/DjangoCompletionSourceBase.cs

```diff
@@ -94,7 +94,7 @@ internal abstract class DjangoCompletionSourceBase : ICompletionSource {
                     "tag"));
             return new FuzzyCompletionSet(
                 "PythonDjangoTags",
-                "Django Tags",
+                Resources.DjangoTagsCompletionSetDisplayName,
```

Comment:
Parameter name says it's a display name, but I'm not sure where it gets displayed, ie. I don't know how to test it.